index.js

```diff
@@ -24,7 +24,16 @@ function validatePass(password) {
     }
```

Comment:
I would rename this commit to added code for checking lower case and uppercase or something along those lines.